compiler/ballerina-parser/src/test/resources/misc/predeclared-module-prefix/predeclared-module-prefix_01.bal

```diff
@@ -0,0 +1,34 @@
+// Override lang.int with custom import
+import ballerina/foo.bar.baz as int;
+
+// Predeclared prefix in annotation and listener decl
+@int:annot
+listener stream:Element myListner = getListner();
+
+type myRecord record {
+    int a;
+    *table:tableRecod;
+};
+
+type myObject object {
+    function foo(int a, any b);
+    *object:customObj;
+};
+
+public function foo() returns int:c {
+    // Using module prefix with predeclared prefix
+    int a = int:sum(1, 2, 3);
```

Comment:
shall we add some tests to cover:
 - global/local var-defs starts with a pre-declared prefix? (eg: `xml:Element x = ...`)
 - Record fields stats with a pre-declared prefix
 - Calling a function from langlibs that is not an attached function (e.g: `pre-declared-prefix:func-name();`)

Reply:
Done. 3rd test case was already there. `int:sum(1, 2, 3);`